lms/static/sass/views/_program-details.scss

```diff
@@ -1,3 +1,77 @@
-.org-logo {
-  width: 120px;
+.program-details {
+	.content-wrapper {
+		max-width: 100%;
+		padding-top: 0;
+	}
 }
+
+.program-header {
+	.banner-content {
+		padding-top: 100px;
+	}
+
+	.title {
+		margin: 0;
```

Comment:
Nitpick: more tab characters here. Let me know if you want to compare editor settings so you don't get these by accident anymore.